key_lists/bgc.yml

```diff
@@ -0,0 +1,38 @@
+keys:
+    # Nutrients
+    N: True  # WOA Nitrate
+    Si: True  # WOA Siliate
+    Iron: True  # Iron
+
+    # Production, Export, Chlorophyll and DMS
+    CHL: True  # Total Chlorophyll
+    # CHD: True  # Diatom Chlorophyll
+    # CHN: True  # Non-diatom Chlorophyll
+    #	DiaFrac: True                  # Diatom Fraction
+
+    IntPP_OSU: True  # OSU Integrated primpary production
+    PP_OSU: True  # OSU Integrated primpary production
+    # DMS_ARAN: True
+    # DMS: True                      
+    # DTC: True                      # Detrital carbon
+    #	LocalExportRatio: True         # Export ratio (no data)
+    #	GlobalExportRatio: True        # Export ratio (no data)
+
+    # Carbonate system:
+    Alk: True  # Glodap Alkalinity
+    DIC: True  # Globap tCO2
+    AirSeaFluxCO2: True  # Air Sea Flux
+    TotalAirSeaFluxCO2: True  # Total global air sea flux
+    pH: True
+
+    # Oxygen:
+    O2: True  # WOA Oxygen
+    TotalOMZVolume: True  # Total Oxygen Minimum zone Volume
+    #	OMZThickness: True             # Oxygen Minimum Zone Thickness
```

Comment:
careful how you format the commented out lines - if someone who doesn't know yaml uncomments these they will immediately run into yaml loading issues coxz the indentation is wrong - I always leave either a single space between # and text or none at all